doc/installationguide.md

```diff
@@ -233,11 +233,12 @@ used for the same purpose. For instance:
     [Cluster](https://nodejs.org/api/cluster.html) module in Node.js and
     [this section](howto.md#iot-agent-in-multi-thread-mode) of the library documentation.
 -   **defaultExpressionLanguage**: the default expression language used to
-    compute expressions, possible values are: `legacy` or `jexl`. When not set or 
+    compute expressions, possible values are: `legacy` or `jexl`. When not set or
     wrongly set, `legacy` is used as default value.
 
--   **explicitAttrs**: if this flag is activated, only provisioned attributes will be processed to Context Broker. 
+-   **explicitAttrs**: if this flag is activated, only provisioned attributes will be processed to Context Broker.
     This flag is overwritten by `explicitAttrs` flag in group or device provision.
+-   **relaxTemplateValidation**: if this flag is activated, `objectIds` for incoming devices are not validated and may include characters such as semi-colons.
```

Comment:
Orion ContextBroker is fine in that case?

Reply:
Good point... `;` is forbidden char in Orion https://fiware-orion.readthedocs.io/en/master/user/forbidden_characters/index.html.

Reply:
But this is `objectId` which is relaxed - the ContextBroker uses the mapped `name` instead.

Reply:
Hi folks, I agree with @jason-fox , the _objectId_ is related to the devices (southbound side), it means it shouldn't be used for the NGSI mapping (northbound side). In a nutshell, we cannot give limitations or constraints to southbound IDs! Obviously, OCB forbidden chars should be taken in care in the _name_ field validation.

Reply:
Maybe that should be remarked in the documenation? Something like this:

```suggestion
-   **relaxTemplateValidation**: if this flag is activated, `objectIds` for incoming devices are not validated and may include characters such as semi-colons. Note that some characters are [forbidden in the CB](https://fiware-orion.readthedocs.io/en/master/user/forbidden_characters/index.html) so if you are using some of them in `objectId` in relax mode, take into account you need to use mappings using `name`s that doesn't have such forbidden chars.
```

Of course other wording expressing the same is also valid.

Reply:
Fixed c92926f - I've reworded and clarified the text.